### DISCUSSION TITLE
Add gifs for the other animated themes on the themes page

### DISCUSSION THREAD

**Discussion by thibaultmol** [JulianPrieber/littlelink-custom#711] [Feature requests]
Currently only 'Stargazer' has a gif but some of the other animated themes should probably also get animated preview images
https://linkstack.org/themes/

**Comment by JulianPrieber** [comment]
Stargazer does not use a GIF background. No themes currently do.
The custom background image does support animated PNGs and maybe even GIFs (I can't remember).

**Reply by thibaultmol** [reply]
Sorry, should have worded better. On that page, which shows all the available themes; only Stargazer has an animated preview of the theme. None of the other animated themes have an animated preview image on that page

**Reply by JulianPrieber** [reply]
Ah, I see.
We had an internal discussion about this with the team after I created the GIF for Stargazer about that exact same topic.
The consensus was that we wanted more GIFs, but no one wanted to do it.

I had to record and create a perfect loop, then convert and optimize.
